index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
+     <nav class="nav">
+         <i id="menu" class="material-icons">menu</i>
+     </nav>
+    </header>
+    <section class="headline">
+        <h1 class="name">Desmond
+            Wanjala</h1>
+        <p class="p-text">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        <button type="button" class="collabo">Start collaboration</button>
+    </section>
```

Comment:
- [ ] As stated in the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly Adjust margin(bottom) for "Headline section" as defined in the design.

Your Project Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/198328019-a64bfa55-5508-404b-abb9-765eaa63e5c6.PNG)

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/198328531-c862e13f-fda2-4951-a935-e91b7aaefede.PNG)